src/alire/alire-publish.adb

```diff
@@ -227,6 +271,146 @@ package body Alire.Publish is
       end;
    end Generate_Index_Manifest;
 
+   ---------------------
+   -- Prepare_Archive --
+   ---------------------
+
+   procedure Prepare_Archive (Context : in out Data) with
+     Pre => Context.Root.Is_Valid;
+
+   procedure Prepare_Archive (Context : in out Data) is
+      use Utils;
+      Target_Dir : constant Relative_Path :=
+                     Paths.Working_Folder_Inside_Root / "archives";
+      Root       : Roots.Root renames Context.Root.Value;
+      Milestone  : constant String :=
+                     TOML_Index.Manifest_File (Root.Release.Name,
+                                               Root.Release.Version,
+                                               With_Extension => False);
+      Git        : constant VCSs.Git.VCS := VCSs.Git.Handler;
+      Is_Repo    : constant Boolean := Git.Is_Repository (Root.Path);
+      Archive    : constant Relative_Path :=
+                     Target_Dir
+                       / (Milestone
+                          & (if Is_Repo
+                             then ".tgz"
```

Comment:
Why is there a difference here? (tgz vs tbz2)

Reply:
bzip2 achieves better ratios, but git doesn't support generating tbz2. We could always use .tgz for consistency.

Reply:
It's not a huge deal, I was just curious.

Reply:
OK then, assuming most people will work from git, they wouldn't notice the difference anyway.